onnxruntime/core/mlas/lib/amd64/QgemvU8S8KernelAvxVnni.asm

```diff
@@ -0,0 +1,385 @@
+;++
```

Comment:
Rather than duplicating code, why not have a parameterized implementation similar to AVX-512 and AVX-512VNNI?

Reply:
This was the initial direction that was considered.
But on further discussion with Microsoft, we decided to keep this flow separate to align with future code design.

Reply:
OK. Hopefully they realize how much code is being duplicated here...